protocol-configuration/pauseState.ts

```diff
@@ -0,0 +1,30 @@
+export const pauseStateConfig: PauseStateConfig = {
```

Comment:
I think we should rethink this test. While it's good to ensure that certain contracts are either paused or not, this config and these types are too specific to PSMs.

Reply:
We could name this in a generic way "state config" and use that to check various state variables. For instance we could add 
```
lusdPSM: {
  paused: true,
  redeemPaused: true,
  surplusTarget: '{bammDeposit}'
}
```
and other state checks

Reply:
I think I prefer this to what we have now, though still not sure if this test is warranted

Reply:
The initial version of this test was done to make sure we don't accidentally unpause/pause something and don't know. We'll loudly get alerted now if we do something that accidentally unpauses. 

If we have need for a more thorough pause state test in the future very down for iterating and breaking out into another PR